### PR TITLE
feat: landing page design refresh

### PR DESCRIPTION
**Pull request by shlomomdahan**
Before:
Screenshot 2024-06-27 at 4 45 45 PM

After: 
Screenshot 2024-06-27 at 4 45 10 PM

### CODE REVIEW

**Comment by lemeurherve** [comment]
I'm not sure this is a good change, we're loosing affordance of the buttons, nothing indicate that each of these lines is a link.

Also, why numbering them? What does it mean? Is it needed?

**Comment by lemeurherve** [comment]
Also, I would not mix #64 with this PR, or at least I would mention that this PR also includes this change.

**Comment by krisstern** [comment]
I am not sure this is a good change either. It looks better and more consistent with the current design of https://www.jenkins.io/ before. 

@gounthar @Vandit1604 What do you think?

**Comment by gounthar** [comment]
It was more pleasing to my terminal-nerd eyes, but that's not a good enough reason. 🥴

**Comment by Vandit1604** [comment]
I get the appeal what @shlomomdahan has in his mind, but this design has some problems like
1. If I visit the site for the 1st time, I won't figure out the buttons immediately. We can style the buttons like this to match the theme of jenkins.io. I think these will suit this site as well.
image

2. We're using `monospace` font as the only font-family on the landing page. I would suggest using `georgia` font for **Jenkins Statistics** and using this font-family (got from devtool on jenkins.io) for the other text.
```
font-family: system-ui, "Segoe UI", roboto, "Noto Sans", oxygen, ubuntu, cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
``` 

WDYT @shlomomdahan @krisstern @gounthar

**Comment by shlomomdahan** [comment]
Thoughts on the page with updated buttons?
Screenshot 2024-06-28 at 10 52 16 AM

Screenshot 2024-06-28 at 10 52 28 AM

**Comment by Vandit1604** [comment]
I would prefer the buttons to not have gradient, but that's my personal preference.
Also, White text on a Blue button has less visibility overall!

**Comment by lemeurherve** [comment]
ICYDK, there is a general Jenkins Design Library that could be used for
inspiration: https://weekly.ci.jenkins.io/design-library

**Comment by krisstern** [comment]
I don't think the buttons should have a color gradient according to our design library.

**Review by krisstern — CHANGES_REQUESTED** [review]
@shlomomdahan Could you please find out what font(s) they are using for [https://www.jenkins.io/](https://www.jenkins.io/) and use that for this website as well? 

Thanks!

**Comment by shlomomdahan** [comment]
@krisstern 

Screenshot 2024-06-30 at 12 03 53 PM

here is comparison with font from Jenkins.io

My opinion is that monospace looks far better.


Screenshot 2024-06-30 at 12 04 54 PM

**Comment by krisstern** [comment]
We understand you have a particular vision for this website, but we would like to make the font family consistent with the current design of [https://www.jenkins.io/](https://www.jenkins.io/) as in https://github.com/jenkins-infra/jenkins.io/blob/458cb120ef73721b3694fe9e52c41b552527aa70/content/stylesheets/core/_base.scss#L5

Please revise or we cannot approve it.

**Comment by krisstern** [comment]
So the `font-family` field should really be:
```javascript
font-family: system-ui, "Segoe UI", roboto, "Noto Sans", oxygen, ubuntu, cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

**Comment by krisstern** [comment]
I think for the blog we current have as the font-family: https://github.com/jenkins-infra/jenkins.io/blob/458cb120ef73721b3694fe9e52c41b552527aa70/content/stylesheets/pages/_blog.scss#L25

So basically you could do for the landing page something similar too.

**Comment by krisstern** [comment]
> I get the appeal what @shlomomdahan has in his mind, but this design has some problems like
> 
> 1. If I visit the site for the 1st time, I won't figure out the buttons immediately. We can style the buttons like this to match the theme of jenkins.io. I think these will suit this site as well.
>    image
> 2. We're using `monospace` font as the only font-family on the landing page. I would suggest using `georgia` font for **Jenkins Statistics** and using this font-family (got from devtool on jenkins.io) for the other text.
> 
> ```
> font-family: system-ui, "Segoe UI", roboto, "Noto Sans", oxygen, ubuntu, cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
> ```
> 
> WDYT @shlomomdahan @krisstern @gounthar @lemeurherve

@Vandit1604 I don't think we are using maybe just `monospace` as the only font-family for the landing page. I see `Georgia` being used as well as least.

**Comment by krisstern** [comment]
So in summary, maybe for the paragraph tags `<p>` (in our case the Material UI Typography equivalents) we should have `font-family: system-ui, "Segoe UI", roboto, "Noto Sans", oxygen, ubuntu, cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";`, but for the headings tag like `<h1>` or their equivalents we should use `font-family: "Georgia", serif;`.

Hope this helps!

**Comment by shlomomdahan** [comment]
@krisstern 

"font-family: system-ui results in Segoe on Windows, San Francisco on macOS, and Roboto on Android." - https://fonts.google.com/knowledge/glossary/system_font_web_safe_font

The font's are all relative to OS

Regardless, I think we should just close this PR then and revert back to how it was before (constant with the Jenkins.io site)

**Comment by krisstern** [comment]
Yes, that is intentional. But I was not the one making that decision for the fonts used for [https://www.jenkins.io/](https://www.jenkins.io/). So please just follow the recommendation(s) that the mentoring team has come up with.

**Comment by krisstern** [comment]
You will have to update the fonts now that we have come to a conclusion as your mentors.

**Comment by shlomomdahan** [comment]
Font's have been updated. I will likewise update PR #64

File: src/pages/landing-page/index.tsx [hunk 1]
```diff
@@ -98,12 +135,10 @@ const LandingPage: React.FC = () => {
                                 },
                             }}
                         >
-                            <Typography variant="h3" sx={{ fontFamily: 'Georgia', fontWeight: 'bold' }}>
+                            <Typography variant="h4" sx={{ fontWeight: 'bold' }}>
```
Comment (krisstern) [inline src/pages/landing-page/index.tsx:138]:
```suggestion
                            <Typography variant="h4" sx={{ fontFamily: 'Georgia, sarif', fontWeight: 'bold' }}>
```

Comment (krisstern) [reply src/pages/landing-page/index.tsx:138]: I think there was a bit of a misunderstanding here

**Review by krisstern — CHANGES_REQUESTED** [review]

**Review by krisstern — APPROVED** [review]

**Pull request merged**